site-src/geps/overview.md

```diff
@@ -0,0 +1,119 @@
+experimental fields.
+
+Before these changes are released, they will also need to be documented.
+GEPs that have not been both implemented and documented before a release
+cut off will be excluded from the release.
+
+### 5. Graduate the GEP to "Standard"
+Once this feature has met the [graduation criteria](/concepts/versioning/#graduation-criteria), it is
+time to graduate it to the "Standard" channel of the API. Depending on the feature, this may include
+any of the following:
+
+1. Graduating the resource to beta
+2. Graduating fields to "standard" by removing `<gateway:experimental>` tags
+3. Graduating a concept to "standard" by updating documentation
+
+## Status
+
+Each GEP has a status field that defines it's current state. Most GEPS will
+proceed through the following states:
+
+* **Provisional:** The goals described by this GEP have consensus but
+  implementation details have not been agreed to yet.
+* **Implementable:** The goals and implementation details described by this GEP
+  have consensus but have not been fully implemented yet.
+* **Experimental:** This GEP has been implemented and is part of the
+  "Experimental" release channel. Breaking changes are still possible.
+* **Standard:** This GEP has been implemented and is part of the
+  "Standard" release channel. It should be quite stable.
+
+Although less common, some GEPs may end up in one of the following states:
+
+* **Deferred:** We do not currently have bandwidth to handle this GEP, it
+  may be revisited in the future.
+* **Rejected:** This proposal was considered by the community but ultimately
+  rejected.
+* **Replaced:** This proposal was considered by the community but ultimately
+  replaced by a newer proposal.
+* **Withdrawn:** This proposal was considered by the community but ultimately
+  withdrawn by the author.
+
+## Format
+
+GEPs should match the format of the template found in [GEP-696](/geps/gep-696).
+
+## Out of scope
+
+What is out of scope: see [text from KEP][kep-when-to-use]. Examples:
+
+* Bug fixes
+* Small changes (API validation, documentation, fixups). It always
+  possible that the reviewers will determine a "small" change ends up
+  requiring a GEP.
+
+## How much additional work is this?
+
+* Need to copy out material into a `.md` document for future
+  reference and keep it up to date.
+* Need to keep GEP up to date with current references to shared doc.
+* Some amount of overhead merging GEPs into the repo.
+
+## FAQ
+
+* Q: Why is it named GEP?
+  * A: To avoid potential confusion if people start following the cross
+    references to the full KEP process.
+* Q: Why have a different process than mainline?
+  * A: We would like to keep the machinery to an absolute minimum for now --
+    this will likely change as we move to v1.
+* Q: Is it ok to discuss using shared docs, scratch docs etc?
+  * A: Yes! We view GEPs as primarily historical record to ensure that any
+    artifacts are preserved for future discussions.
```

Comment:
I'm actually against this. I can't think of a case where it wouldn't be better to simply say less up front with a small initial GEP explaining the "What" and "Why" at a basic level before moving on to any implementation details, but "scratch docs" in google docs and so forth is often a sprawl of technical details before we've even formally agreed that there is a problem, what that problem is, and that we want to fix it.

I'm not going to block this PR on this language, but I am going to be grumpy about it and try to persuade others that we should _pull away_ from the google docs pre-process we have today and stick to an iterative GEP process instead.

Reply:
The GitHub diff is weird here as this is one of the few things I didn't touch from the original `contributing/geps` page. I agree that we have to be careful with this approach. I do think shared docs are a useful tool in some cases that can help simplify the initial brainstorming process and get past some of the limitations of PR reviews, but I want to be very clear that we need to represent that context and major discussion points in the GEP so it does not require searching around through old docs to understand why decisions were made. Hopefully this update makes sense, PTAL.

Reply:
LGTM, no blockers.